### PR TITLE
Stop bundling libraries in release

### PR DESCRIPTION
**Pull request by nhooyr**
- Instead we now use CentOS 7 for the static build to guarantee
that we only depend on libc v2.17

- For macOS we now pull in a static node binary and bundle that instead.

Updates #1738

### CODE REVIEW

File: ci/images/centos7/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+RUN yum groupinstall -y 'Development Tools'
+RUN yum install -y python2 libsecret-devel libX11-devel libxkbfile-devel
+
+RUN npm config set python python2
+
+RUN yum install -y epel-release && yum install -y jq
+RUN yum install -y rsync
+
+# Copied from ../Dockerfile
```
Comment (code-asher) [inline ci/images/centos7/Dockerfile:15]: I think this may mean to point to `../debian8/Dockerfile`.

**Comment by code-asher** [comment]
You probably already saw the build error, but I'll make note there's a missing `/` just in case.

**Comment by code-asher** [comment]
Er, I mean `\`.

**Comment by nhooyr** [comment]
Mhm yea, pushing rn.

**Comment by meain** [comment]
Are you planning on making a release with this fix on top of 3.4.0 soon?

**Comment by nhooyr** [comment]
Yes 3.4.1 should be out today.

**Comment by nhooyr** [comment]
New release is out!

https://github.com/cdr/code-server/releases/tag/v3.4.1

**Pull request merged**